maven-modules/pom.xml

```diff
@@ -24,6 +24,7 @@
         <module>maven-profiles</module>
         <module>versions-maven-plugin</module>
         <module>version-collision</module>
+        <module>maven-properties</module>
```

Comment:
We like to keep these alphabetical, so please move this line up a bit so that it's after maven-profiles but before versions-maven-plugin